lib/quicly.c

```diff
@@ -3330,7 +3330,7 @@ static int handle_ack_frame(quicly_conn_t *conn, size_t epoch, quicly_ack_frame_
     /* OnPacketAcked and OnPacketAckedCC */
     if (bytes_acked > 0) {
         quicly_cc_on_acked(&conn->egress.cc, (uint32_t)bytes_acked, frame->largest_acknowledged,
-                           conn->egress.sentmap.bytes_in_flight + bytes_acked);
+                           (uint32_t)(conn->egress.sentmap.bytes_in_flight + bytes_acked));
```

Comment:
Our PN size is 16 bits and MTU is 1280 bytes, therefore I believe it's impossible to go above 4GB of inflight data.